src/hooks/core/useCoreChatSystemHook.ts

```diff
@@ -7,6 +7,7 @@ import AnonymousMessageRepo from '../../service/repo/anonymousMessageRepo';
 import ChannelList from '../../database/schema/ChannelListSchema';
 import ChatSchema from '../../database/schema/ChatSchema';
 import SignedMessageRepo from '../../service/repo/signedMessageRepo';
+import StorageUtils from '../../utils/storage';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `StorageUtils` is added but it seems like it's not used anywhere in the provided code. If it's not used in the rest of the file, consider removing this import to keep the code clean and maintainable.
```diff
- import StorageUtils from '../../utils/storage';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->